components/DocumentList.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import type {
+  DocumentListIAllDocumentModels,
+  IAllDocumentModels,
+} from "@bcc-code/bmm-sdk-fetch";
+
+const props = defineProps<{
+  list: DocumentListIAllDocumentModels | null;
+  pending: boolean;
+}>();
+
+watch(
+  props,
+  () => {
+    toolbarTitleStore().setToolbarTitle(props.list?.title ?? " ");
+  },
+  { immediate: true },
+);
+</script>
+
+<template>
+  <div>
+    <template v-if="props.pending">
+      <ul>
+        <li
+          v-for="index in 5"
+          :key="index"
+          class="my-6 h-11 w-full animate-pulse rounded-lg bg-background-2"
+        ></li>
+      </ul>
+    </template>
+    <template v-else>
+      <DocumentModels
+        v-if="
+          list !== null &&
+          list !== undefined &&
+          list.items !== null &&
+          list.items !== undefined
+        "
+        :models="list.items as IAllDocumentModels[]"
```

Comment:
Type assertion should not be necessary here. Why was it used?

_Note:_ Type assertion should be avoided whenever possible. See https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions - you'll loose all benefit of a type-safe system. This is valid Typescript:
```ts
const foo = 123;
const bar: string = foo as any as string;
```

Reply:
fixed.